kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/Kin.java

```diff
@@ -184,21 +197,21 @@ private static void checkInstanceNotNull() throws ClientException {
 	 * Launch Kin Marketplace if the user is activated, otherwise it will launch Welcome to Kin page.
 	 *
 	 * @param activity the activity user can go back to.
-	 * @throws ClientException
 	 */
 	public static void launchMarketplace(@NonNull final Activity activity) throws ClientException {
 		checkInstanceNotNull();
 		instance.eventLogger.send(EntrypointButtonTapped.create());
 		boolean isActivated = AuthRepository.getInstance().isActivated();
-		if (isActivated) {
+		boolean isAccountCreated = instance.accountManager.getAccountState() == AccountManager.CREATION_COMPLETED;
```

Comment:
isAccountCreated() should be a method in accountManager (hide AccountManager.CREATION_COMPLETED sate as much as possible)